pydev debugger: process 11105 is connecting

Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Volumes/Cyber/CyberPatriots/my_scripts_and_STIGS/ScriptCyber/GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripLINUXONLY.py", line 5, in <module>
    import distro  # for figuring out what linux distro
ModuleNotFoundError: No module named 'distro'
